langgraph/src/pregel/index.ts

```diff
@@ -601,10 +579,8 @@ function _prepareNextTasks(
           }
         });
 
-        // skip if condition is not met
-        if (proc.when === undefined || proc.when(val)) {
-          tasks.push([proc, val, name]);
-        }
+        tasks.push([proc, val, name]);
```

Comment:
I don't know if this is valid, but all of the existing tests are passing.

Reply:
why would it not be valid?

Reply:
I was just comparing with the current Python implementation which relies on the `for_execution` API, which isn't implemented yet in JS.

https://github.com/langchain-ai/langgraph/blob/main/langgraph/pregel/__init__.py#L1287